### DISCUSSION TITLE
Start nvim and report an error

### DISCUSSION THREAD

**Discussion by ConanEdogawa1** [LazyVim/LazyVim#303] [Q&A]
Lua module not found for config of bufferline.nvim. Please use a `config()` function instead

**Comment by folke** [comment]
The bufferline plugin probably wasn't installed properly due to internet issues.

Remove the plugin directory and best to set `config.concurrency=10` for lazy.nvim. Check the docs

**Comment by ConanEdogawa1** [comment]
how to set this

**Reply by folke** [reply]
As I said. Read the lazy.nvim docs. The file you'll have to change is `lua/config/lazy.lua` It's all in the LAzyVim and lazy.nvim docs. Read them

**Reply by ConanEdogawa1** [reply]
ok,Thanks. I'm bothering you
